adlfs/transfer.py

```diff
@@ -217,11 +215,9 @@ class ADLTransferClient(object):
     adlfs.multithread.ADLUploader
     """
 
-    DEFAULT_TMP_PATH = 'tmp'
-
     def __init__(self, adlfs, name, transfer, merge=None, nthreads=None,
-                 chunksize=2**28, blocksize=2**25, tmp_path=DEFAULT_TMP_PATH,
-                 tmp_unique=True, persist_path=None, delimiter=None):
+                 chunksize=2**28, blocksize=2**25, chunked=True,
```

Comment:
I think it is ok to include the ability to tweak these things, but yes, we should default everything to "optimal" performance and usability from our perspective. That way, if a user wants to change it, its assumed they know what they are doing and have a good reason.